hash/sega_beena_cart.xml

```diff
@@ -85,21 +256,48 @@ license:CC0
+				<rom loadflag="load16_word_swap" name="S-100006-1000 - Kazoku Minna no Nouryoku Trainer (Japan).bin" size="0x00400000" crc="8097f3ec" sha1="1e65579b74f141591db2122b34ce293d1d2120ff"/>
 			</dataarea>
 			<dataarea name="pages" size="0x2000000"> <!-- book pages -->
-				<rom name="0 - 0001.png" size="0x01e37832" crc="8de70b0f" sha1="d198caa20060f49774c2e2103fe1ef168f16e13a" />
-				<rom name="0 - 0002.png" size="0x01d7eeb6" crc="c8677ffc" sha1="85419e7ae1f6f85659ab848855be4db192b20e21" />
-				<rom name="0 - 0003.png" size="0x01e24e0f" crc="ffd4762a" sha1="9a84422272bed23c77392031c1b858c6ec5e2d1a" />
-				<rom name="0 - 0004.png" size="0x01d465df" crc="cb60e2d6" sha1="5324a22fdde1e24f8797cba51e9589f3b81d3e41" />
-				<rom name="0 - 0005.png" size="0x01b9db20" crc="fabd3a8f" sha1="255574ae14304d9068029189aeb722357a257c5c" />
-				<rom name="0 - 0006.png" size="0x01d98d45" crc="f560da1d" sha1="857ba8882146b2cf8c35b17fb66711310741f52d" />
-				<rom name="0 - 0007.png" size="0x01a0648c" crc="6b83e144" sha1="b33cd0ba66270ca99661b28eab5892e21ac42b2f" />
-				<rom name="0 - 0008.png" size="0x01dfa9e0" crc="3fd345d6" sha1="8ef21ccfa7dd473f269996bf5de8c7b7c320cd88" />
-				<rom name="0 - 0009.png" size="0x01124676" crc="b1a8e3cc" sha1="231048de3bc7863e47a8b5da79a8a5bed501d7b6" />
-				<rom name="0 - 0010.png" size="0x01e618a8" crc="571593da" sha1="4125714a58e526b74e7c2e60294bb6450c2bdc8c" />
+				<rom name="0 - 0001.png" size="0x01e37832" crc="8de70b0f" sha1="d198caa20060f49774c2e2103fe1ef168f16e13a"/>
+				<rom name="0 - 0002.png" size="0x01d7eeb6" crc="c8677ffc" sha1="85419e7ae1f6f85659ab848855be4db192b20e21"/>
+				<rom name="0 - 0003.png" size="0x01e24e0f" crc="ffd4762a" sha1="9a84422272bed23c77392031c1b858c6ec5e2d1a"/>
+				<rom name="0 - 0004.png" size="0x01d465df" crc="cb60e2d6" sha1="5324a22fdde1e24f8797cba51e9589f3b81d3e41"/>
+				<rom name="0 - 0005.png" size="0x01b9db20" crc="fabd3a8f" sha1="255574ae14304d9068029189aeb722357a257c5c"/>
+				<rom name="0 - 0006.png" size="0x01d98d45" crc="f560da1d" sha1="857ba8882146b2cf8c35b17fb66711310741f52d"/>
+				<rom name="0 - 0007.png" size="0x01a0648c" crc="6b83e144" sha1="b33cd0ba66270ca99661b28eab5892e21ac42b2f"/>
+				<rom name="0 - 0008.png" size="0x01dfa9e0" crc="3fd345d6" sha1="8ef21ccfa7dd473f269996bf5de8c7b7c320cd88"/>
+				<rom name="0 - 0009.png" size="0x01124676" crc="b1a8e3cc" sha1="231048de3bc7863e47a8b5da79a8a5bed501d7b6"/>
+				<rom name="0 - 0010.png" size="0x01e618a8" crc="571593da" sha1="4125714a58e526b74e7c2e60294bb6450c2bdc8c"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="kouchuu" supported="no">
+		<description>Kouchuu Ouja Mushiking: Mori no Tami no Densetsu: Minna de Tanken! Kouchuu no Mori</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="甲虫王者ムシキング ～森の民の伝説～ みんなでたんけん!甲虫の森"/>
```

Comment:
This has a halfwidth exclamation mark in it.

Reply:
Replaced all halfwidth with regular chars and removed spaces!